### ISSUE TITLE
awaitViewModelStart not awaiting

### ISSUE THREAD

**Issue by asapha** [copper-leaf/ballast#46]
Unless I'm missing something, the current implementation of `awaitViewModelStart` doesn't do anything, as `take` isn't a terminal operator.

```kotlin
/**
 * Suspend until the ViewModel has started
 */
public suspend inline fun <Inputs : Any, Events : Any, State : Any> Flow<BallastNotification<Inputs, Events, State>>.awaitViewModelStart() {
    filter {
        it is BallastNotification.ViewModelStatusChanged<Inputs, Events, State> && it.status == Status.Running
    }.take(1)
}
```

**Comment by cjbrooks12** [comment]
Oh dang, you're right, nice catch. Looks like I accidentally switched it from `first()` to `take(1)` with the 3.0.0 release. I'll get a fix released shortly

**Comment by cjbrooks12** [comment]
This should be fixed now in [v3.0.2](https://github.com/copper-leaf/ballast/releases/tag/3.0.2)
